docs/shared/android/configure_java.md

```diff
@@ -21,15 +19,15 @@ That's it! To verify correctness, open a new shell, navigate to a directory with
 
 This works because macOS comes with a `/usr/bin/java` stub which will defer the location of the JDK to the value in the `JAVA_HOME` environment variable.
 
-### Method #2: install Java 11 from Homebrew
+### Method #2: install Java 17 from Homebrew
 You can install Java from [Homebrew]([url](https://brew.sh/)) using the command below.
 
 ```bash
-brew install java11
+brew install openjdk@17
 ```
 
 ### Method #3: install from website
-TODO
+See https://docs.oracle.com/en/java/javase/17/install/installation-jdk-macos.html
```

Comment:
Not sure if this was the intended info for this method.

Reply:
oops